fill_esd_log.py

```
@@ -418,85 +429,203 @@ def main() -> None:
+                                contact_span.set_attribute("enrichment.success", True)
+                                c["address"] = contact_info["address"]
+                                c["city"] = contact_info["city"]
+                                c["state"] = contact_info["state"]
+                                c["website_or_email"] = contact_info["website_or_email"]
+                                c["phone"] = contact_info["phone"]
+
+                                # Record success metric
+                                contact_enriched_counter.add(
+                                    1, {"business_name": business_name}
+                                )
+                        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)